k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml

```diff
@@ -40,3 +40,4 @@
     "sha256:d86e162632fba60a0f1575a396c78574b9f0472ff8fae343fe4fda8bc4e1aba0": ["v1.3.4"]
     "sha256:53b62ec56ded588d3f7c31b2fbe290e03d0cbbaf4566d0a675216aeea9c74ec4": ["v1.3.5"]
     "sha256:da4dfe795a2695e8786942180d39c6141b6667901ad844e8967786e6d5239e34": ["v1.3.9"]
+    "sha256:25e0acc00b79176f27919f0bcf5e27f0d8d481909b83ce5b72d16e37df988e5d": ["v1.8.1"]
```

Comment:
this should be under `gcp-compute-persistent-disk-csi-driver` instead of `gcp-filestore-csi-driver`

Reply:
gah thx that's why we do reviews